src/tables/cmap/mod.rs

```diff
@@ -180,6 +180,54 @@ impl<'a> Subtable<'a> {
             None
         }
     }
+
+    /// Calls `f` for all codepoints contained in this subtable.
+    ///
+    /// This is a low-level method and it doesn't check that the current
+    /// encoding is Unicode. It simply calls the function `f` for all `u32`
+    /// codepoints that are present in this subtable.
+    ///
+    /// Note that this may list codepoints for which `glyph_index` still returns
+    /// `None` because this method finds all codepoints which were _defined_ in
+    /// this subtable. The subtable may still map them to glyph ID `0`.
+    ///
+    /// Returns without doing anything:
+    /// - when format is `MixedCoverage`, since it's not supported.
+    /// - when format is `UnicodeVariationSequences`, since it's not supported.
+    pub fn codepoints<F>(&self, f: F)
+    where
+        F: FnMut(u32),
```

Comment:
I think this can be inlined, e.g. `pub fn codepoints<F: FnMut(u32)>(&self, f: F)`